### ISSUE TITLE
Include a WebDAV server

### ISSUE THREAD

**Issue by ad-si** [ad-si/TaskLite#67]
Starts a WebDAV server which allows accessing the tasks via a WebDAV client (e.g. Apple Finder or Cyberduck)

**Comment by xmailla** [comment]
I really like this idea. Is there any plan to document the best way to sync ? Currently I am doing it via watchexec (a rust binary that exec a command whenever something has changed/been modified but that' not totally satisfying IMO)

**Comment by ad-si** [comment]
Yeah, the sync story so far isn't really thought-out. From the documentation at https://tasklite.org/roadmap.html?highlight=sync#syncing:

> There is no concrete plan for native synchronization support yet. I've been using Dropbox for synchronizing the TaskLite database among several computers, which has worked astonishingly well so far.

> Another good solution could be [Litestream](https://litestream.io/), which continuously streams SQLite changes to S3-compatible storage.
